opencog/nlp/learn/README

```diff
@@ -286,35 +286,31 @@ alpha-pages after all that work)
     cp -pr alpha-pages beta-pages
 ```
 * Review the file `opencog/nlp/learn/run/README`, and then copy the
-  scripts into your working dir. 
+  scripts into your working dir, if you didn't do it before.
 * Modify as needed for chosen language. Most of them don't need modification,
   by default, unless you want to do something unusual.
 
-  -- `run-all-servers.sh` runs the cogserver for
-     one language, in a multi-paneled `byobu` window.
-     It doesn't start the final batch script `./wiki-ss-<lang>.sh`,
-     which needs to be started by hand (for the <lang> that you want to
-     process). Use F4 to move to the 'parse' tab and start it there.
-
```

Comment:
what happened to run-all-servers.sh?  I can see you trying to o it with just once script below, but you still need to have a byobu window so that you can control processes all at once.   I've been running with 3 or 4 or 5 virtual machines, each running different stages of the pipeline, ad you can just do this all in just one window -- you need multiple windows, one or two for each VM, and then have byobu in each window, just so the whole thing doesn't spiral out of control.   Also, some of these may be working on different languages. ... the ability to monitor progress is very important.